tests/classes/test_reportviews.py

```diff
@@ -116,6 +116,17 @@ def test_edge_members(edgelist3):
     assert H.edges.members(0) == [1, 2, 3]
     assert H.edges.members() == [[1, 2, 3], [3, 4], [4, 5, 6]]
     assert H.edges.members(dtype=dict) == {0: [1, 2, 3], 1: [3, 4], 2: [4, 5, 6]}
+    with pytest.raises(XGIError) as exception_info:
+        H.edges.members(dtype=np.array)
+    assert "Unrecognized dtype" in str(exception_info.value)
+
+    with pytest.raises(XGIError) as exception_info:
+        H.edges.members(slice(1, 4, 1))
+    assert "try list(H.edges)[1:4:1]" in str(exception_info.value)
+
+    with pytest.raises(IDNotFound) as exception_info:
+        H.edges.members("test")
+    assert "Item test not in this view" == str(exception_info.value)
```

Comment:
I'm not sure what I think about these ones. What is the benefit of testing that "Item test not in this view" is exactly the message attached to the exception versus just testing that the type of the exception is correct?

Reply:
Usually one relies on the type of the exception to test what kind of error was raised. In he first two cases, the actual exception type raise should be ValueError and not really XGIError, and testing for that would be better than testing for the actual message I think.